gemini/use-cases/retrieval-augmented-generation/code_rag.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>grammer → grammar</p><p> </p><p>Why do we need to download this from the repository? Does pip installing treesitter not include this?</p><p> </p><p>You should also be able to download a single file instead of the full repository, if you just need one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Corrected.</p><p> </p><p>Tree Sitter library and the grammar used to create the compiled object are separate entities. The official documentation mentions the same steps that I had followed in the notebook. So it wont be possible to do the step with a single file.</p><p> </p><p>Ref: <a href="https://github.com/tree-sitter/py-tree-sitter" target="_blank">https://github.com/tree-sitter/py-tree-sitter</a></p>